lib/commands/update.js

```diff
@@ -49,8 +53,20 @@ module.exports = Command.extend({
     var sort = require('sort-package-json');
 
     return migrate(fromTag, toTag, packageJSON, isAddon, this.ui).then(function(obj) {
-      console.log(chalk.dim(JSON.stringify(sort(obj), null, 2)));
+      var sortedPackageJSON = JSON.stringify(sort(obj), null, 2);
+
+      console.log(chalk.dim(sortedPackageJSON));
       console.log("\n");
+
+      fs.writeFileSync(
+        packageJSONPath,
+        sortedPackageJSON,
+        'utf8'
+      );
+
+      console.log("Running yarn install...");
+
+      return RSVP.resolve(execa('yarn'));
```

Comment:
The user can be using npm to manage dependencies. ember-cli uses npm unless yarn.lock file exists, in that case it uses yarn.